frontend_next/components/caseStudy/MyInfo.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from "next/image";
+
+import { myInfo } from "@/constants";
+
+const MyInfo = () => {
+  return (
+    <div className="space-y-9 md:space-y-20">
+      {myInfo.map((info) =>
+        info.label === "My Socials" ? (
+          <div key={info.label} className="space-y-3 md:space-y-7">
+            <h2 className="infoTitle">{info.label}</h2>
+            <div className="flex gap-4">
+              {info.icons?.map((icon) => (
+                <Image
+                  key={icon.name}
+                  src={icon.icon}
+                  alt={`social icon: ${icon.name}`}
+                  width={0}
+                  height={0}
+                  className="infoIcons"
+                />
+              ))}
+            </div>
+          </div>
+        ) : (
+          <div key={info.label} className="space-y-3 md:space-y-7">
+            <h2 className="infoTitle">{info.label}</h2>
+            <div className="flex items-center gap-2">
+              <Image
+                src={info.icon}
+                alt="contact icon"
+                width={0}
+                height={0}
```

Comment:
It's important for the browser when rendering the image to know the size of the images. Have you ever been on a website and as its loading all the elements are shifting around and moving? This can be a cause of that. Set the correct dimensions so the browser knows what space to make for the element as it loads in.